### DISCUSSION TITLE
Azure Service for cloudflare cdn replacement or other solution for blank domain

### DISCUSSION THREAD

**Discussion by Timsto** [microsoft/AzUrlShortener#446] [Ideas]
Hey, 

is there any nativ azure service to create the same functions as described with Cloudflare? 
that i can type domain.net/blog and will be directly redirected and not need to go with domain.net/api/urlredirect/blog?

**Comment by fboucher** [comment]
That would be great!
I'm not aware of this kind of service, however.

**Comment by fs366e2spm** [comment]
Well, I think Azure API Management (APIM) could be a way. Worth to try. URL transformation is possible and the domain can be registered in APIM as well.

**Reply by fboucher** [reply]
yes indeed!
In fact, when I started working on v3 I did consider APIM and the cost was low too. I didn't keep it because, at the time, I was trying to have only the Azure Static WebApp... But now that we have a standalone function, this could be looked at.
